source/query-with-natural-language/enable-natural-language-querying.txt

```diff
@@ -69,9 +68,20 @@ Steps
            - In the top menu bar, click :guilabel:`MongoDB Compass`.
            - ``⌘`` + ``,``
 
+   .. step:: Enable the AI Features Setting
+
+      You only need to enabled AI features if it is your first time 
+      using natural language querying.
+
+      a. Select :guilabel:`Artificial Intelligence`.
+      #. Click the :guilabel:`Enable AI Features` check box.
+      #. Click :guilabel:`Save`.
```

Comment:
The behavior was changed recently (try Compass 1.42.6-beta.2), so the Atlas login card appears immediately after clicking on the "Enable AI Features" checkbox. But we would like to point out that users should save settings before or after logging in to apply changes and enable GenAI. This limitation exists until we implement autosving settings (see [COMPASS-7866](https://jira.mongodb.org/browse/COMPASS-7866)).

Reply:
Ok I understand the change and tested with 1.42.6-beta2, I have simplified the procedure to reflect the updated settings functionality @alenakhineika

Reply:
I think you miss now the `#. Click :guilabel:`Save`.` AFTER logging in with Atlas. A user still needs save settings, otherwise, they will be logged in with Atlas, but when they go to the query or aggregation page, they won't see the GenAI features.

Reply:
Updated and I noticed the message changed slightly to `You can create queries and aggregations with generative AI.` so I added that here as well.